src/core/kinesis-firehose-defaults.ts

```diff
@@ -0,0 +1,11 @@
+import { Compression, S3BucketProps } from '@aws-cdk/aws-kinesisfirehose-destinations-alpha';
+import { Duration, Size } from 'aws-cdk-lib';
+
+export function DefaultDestinationsS3BucketProps(_dataOutputPrefix?: string) {
```

Comment:
Minor: Why is `Default` capitalized? I thought the convention in Typescript was that functions are camelCase rather than PascalCase. We'll also need to figure out how to enforce these conventions. I think this [rule](https://typescript-eslint.io/rules/naming-convention/) should be good. Obviously this can be a separate PR.

Reply:
Yep, not sure why they are doing [that](https://github.com/awslabs/aws-solutions-constructs/blob/39b897d653eb1d40acd5d6b7e9c778568811f1fc/source/patterns/%40aws-solutions-constructs/core/lib/kinesis-firehose-s3-defaults.ts#L17). Perhaps because it's supposed to mimick a Props interface. But I agree, I will switch to lower case